docs/how-to/websites-on-ipfs/gateway-redirects.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Gateway redirects
+description: What gateway redirects are and how to use them with a website on IPFS.
+---
+
+# THIS IS A DRAFT. DO NOT MERGE.
+
+# To do:
+1. How to redirect old URL to a new place (301 and 302 redirects)
+1. How to use it for PWA/SPA hosting (catch-all 200) 
+1. How to use it to provide custom `404 not found` pages (superceding what `ipfs-404.html` does - ideally not mention old way)
```

Comment:
:+1: :100:  this is the meat of the end-user document – Examples section should be prominent, allowing for learnign by example, instead of reading dry specs :)